plugins/woocommerce/src/Database/Migrations/CustomOrderTable/CLIRunner.php

```diff
@@ -1063,4 +1064,73 @@ function( $key, $hpos_value, $cpt_value ) {
 			array( 'property', 'hpos', 'post' )
 		);
 	}
+
+	/**
+	 * Backfills an order from either the HPOS or the posts datastore.
+	 *
+	 * ## OPTIONS
+	 *
+	 * <id>
```

Comment:
`<order id>` would be clearer here.

Reply:
I've applied this suggestion (also to another command where I was using `<id>`).